hack/windows/Install-OVS.ps1

```diff
@@ -63,20 +73,29 @@ function CheckIfOVSInstalled() {
 }
 
 function DownloadOVS() {
-    If (!(Test-Path $OVSDownloadDir)) {
-        mkdir -p $OVSDownloadDir
-    }
-    Log "Downloading OVS package from $OVSDownloadURL to $OVSZip"
-    curl.exe -sLo $OVSZip $OVSDownloadURL
-    If (!$?) {
-        Log "Download OVS failed, URL: $OVSDownloadURL"
-        exit 1
+    if ($LocalFile -ne "") {
+        Log "Use local file: $LocalFile"
+        return
+    } else {
```

Comment:
do you need the `else` here, given that there is a `return` statement in the `if` block?

Reply:
Exactly, it can be removed. Done.